include/picongpu/plugins/output/WriteSpeciesCommon.hpp

```diff
@@ -37,7 +37,7 @@ namespace picongpu
         {
             using type = typename pmacc::traits::Resolve<T_Type>::type::type;
 
-            bool isMappedMemorySupported = alpaka::hasMappedBufSupport<::alpaka::Pltf<cupla::AccDev>>;
+            bool isMappedMemorySupported = alpaka::hasMappedBufSupport<::alpaka::Platform<cupla::AccDev>>;
```

Comment:
since I did not read the alpaka changes - this renaming is correct @psychocoderHPC, right?

Reply:
Yes this is one of the breaking changes of alpaka